src/Utilities/mockoptimizer.jl

```diff
@@ -389,13 +395,14 @@ The primal values of the variables in the order returned by `ListOfVariableIndic
 If `termstatus` is missing, it is assumed to be `MOI.OPTIMAL`.
 If `primstatus` is missing, it is assumed to be `MOI.FEASIBLE_POINT`.
 If `dualstatus` is missing, it is assumed to be `MOI.FEASIBLE_POINT` if there is a primal solution and `primstatus` is not `MOI.INFEASIBLE_POINT`, otherwise it is `MOI.INFEASIBILITY_CERTIFICATE`.
+The bases status are set to the status specified by `conbasis`. Each pair is of the form `(F,S)=>[...]` where `[...]` is the the vector of basis status for the constraints `F`-in-`S` in the order returned by `ListOfConstraintIndices{F,S}`.
```

Comment:
The function signature in the docstring above doesn't include `conbasis`, so what does this refer to?